JASP-Desktop/html/js/utils.js

```diff
@@ -372,8 +372,29 @@ function formatColumn(column, type, format, alignNumbers, combine, modelFootnote
                 isNumber = true
             }
             else {
-                let _content = (html) ? "&minus;" : "-";
-                formatted = { content: content.toFixed(dp).replace(/-/g, _content), "class": "number" }
+				var strContent;
+
+				if (content < 1/(Math.pow(10,dp))) {
+					let _sign = (html) ? "&minus;" : "-";
+					var exponentiated = content.toExponential(dp).replace(/-/g, _sign)
+
+					var split = exponentiated.split("e")
+					var mantissa = split[0]
+					var exponent = split[1]
+					var exponentSign = exponent.substr(0, 1)
+					var exponentNum = exponent.substr(1)
+
+					if (html) {
+						strContent = mantissa + "e&thinsp;" + exponentSign + exponentNum;
+					} else {
+						strContent = mantissa + "e" + exponentSign + exponentNum;
+					}
+
+				} else {
+					let _content = (html) ? "&minus;" : "-";
+					strContent = content.toFixed(dp).replace(/-/g, _content)
+				}
+				formatted = { content: strContent, "class": "number" }
```

Comment:
Can you fix the indentation?

Reply:
Do you mean that I use tab and the rest of this files uses spaces for indentation? The rest of our code uses tab, so I can transform all the spaces in tabs. Or do you mean something else?

Reply:
Yeah I mean making the indentation consistent within the file.
If you make everything tabs can you do it in a separate commit? So we can see the actual changes you made in the future